sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -0,0 +1,3 @@
+.v-btn {
```

Comment:
The idea of creating vuetify.scss is to create all styles which involves overriding the vuetify here right?

Reply:
Yes, it is.

Reply:
We can add this point to wiki if that will helpful.

Reply:
Well, we can. We do not have any other class or guide similar to this right now.